### ISSUE TITLE
Failed to execute 'addEventListener' on 'EventTarget'

### ISSUE THREAD

**Issue by grdit** [mikitex70/redmine_drawio#96]
Following error occurs when attaching an XML drawing on a Wiki page:

`{{drawio_attach(File.xml,hilight=#0000ff)}}`

`Failed to execute 'addEventListener' on 'EventTarget': parameter 2 is not of type 'Object'.`

```
Environment:
  Redmine version                4.1.1.stable
  Ruby version                   2.6.6-p146 (2020-03-31) [x86_64-linux]
  Rails version                  5.2.4.2
  Environment                    production
  Database adapter               Mysql2
  Mailer queue                   ActiveJob::QueueAdapters::AsyncAdapter
  Mailer delivery                smtp
SCM:
  Git                            2.25.1
  Filesystem                     
Redmine plugins:
  additionals                    3.0.2
  redmine_agile                  1.6.1
  redmine_checklists             3.1.18
  redmine_contacts               4.3.3
  redmine_contacts_helpdesk      4.1.11
  redmine_drawio                 1.1.0
  redmine_drive                  1.1.1
  redmine_favorite_projects      2.1.1
  redmine_issue_dynamic_edit     0.7.2
  redmine_more_previews          2.0.3
  redmine_people                 1.6.0
  redmine_xlsx_format_issue_exporter 0.1.6
  redmine_zenedit                2.0.0
  redmineup_tags                 2.0.8
  view_customize                 2.8.1
```

Tried with different browsers, same thing.

**Comment by mikitex70** [comment]
Hi @grdit,
I've pushed a fix in the `develop` branch. Could you give it a try to confirm it is working fine?

**Comment by grdit** [comment]
Hello @mikitex70,
it works fine now.
Thanks a lot, awesome job!

**Comment by mikitex70** [comment]
Fix officially released in the `master` branch as version `v1.1.1`.
I close this issue.
Thanks for reporting it.
